### ISSUE TITLE
RTM API not available for modern scoped apps

### ISSUE THREAD

**Issue by tt92** [travisghansen/node-red-slack#29]
Hey there,

are there any plans to move away from Slack's [Real Time Messaging API](https://slack.dev/node-slack-sdk/rtm-api) (and use the Events or Web API instead?

Currently, the Node-RED nodes are not compatible with modern scoped apps and only run with "classic"/legacy apps.

At least the documentation could be extended to provide some guidance:

- create a "classic" app via https://api.slack.com/apps?new_classic_app=1
- create a "legacy bot user" (and **don't** "update scopes")

Cheers,

TT

**Comment by travisghansen** [comment]
Yeah it needs better documentation about the API requirements for sure. You can use the web API 100% but it simply uses the same bot token as the rtm aspect.

I don't see moving from RTM for this package honestly but if others want to chime in I'm listening. This is a full blown bot as it stands. If needs are limited creating simple http in (events API) and http out (web API) seems sufficient.
